lua/terrortown/menus/gamemode/base_gamemodemenu.lua

```diff
@@ -1,11 +1,16 @@
 ---
 -- @class CLGAMEMODEMENU
 
+local TryT = LANG.TryTranslation
+local lowerString = string.lower
+local findString = string.find
```

Comment:
you don't have to change this, but I'd recommend renaming those to `stringLower` and `stringFind` respectively

Reply:
I agree, only later saw the style you use for this. Just leaving the dot out.